Makefile

```diff
@@ -1102,22 +1102,18 @@ else
 	@echo Cannot run an astyle check, your system either does not have astyle, or it is too old.
 endif
 
-JSON_FILES = $(shell find data -name "*.json" | sed "s|^\./||")
-JSON_WHITELIST = $(filter-out $(shell cat json_blacklist), $(JSON_FILES))
-
-style-json: $(JSON_WHITELIST)
-
-$(JSON_WHITELIST): json_blacklist json_formatter
+style-json: json_blacklist $(JSON_FORMATTER_BIN)
 ifndef CROSS
-	@$(JSON_FORMATTER_BIN) $@
+	find data -name "*.json" -print0 | grep -v -z -F -f json_blacklist | \
+	  xargs -0 -L 1 $(JSON_FORMATTER_BIN)
 else
 	@echo Cannot run json formatter in cross compiles.
 endif
 
-style-all-json: json_formatter
+style-all-json: $(JSON_FORMATTER_BIN)
 	find data -name "*.json" -print0 | xargs -0 -L 1 $(JSON_FORMATTER_BIN)
 
-json_formatter: $(JSON_FORMATTER_SOURCES)
+$(JSON_FORMATTER_BIN): $(JSON_FORMATTER_SOURCES)
```

Comment:
I totally forgot it would break building of json formatter on Jenkins :(

It was previously called with `make json_formatter` to build only json formatter.

Reply:
I guess will replace it with `make style-all-json` for now, but that would fail builds until I fix all that `data/names`-jsons (e.g. https://ci.narc.ro/job/Cataclysm-Doxygen/10256/console).

Reply:
Ah, I was worried that might happen, but I checked the build scripts and they weren't using this target.

Easiest solution is to add
```make
json_formatter: $(JSON_FORMATTER_BIN)
```